src/main/resources/application-dev.yaml

```
@@ -9,7 +9,7 @@ spring:
     defer-datasource-initialization: false
 
   datasource:
-    url: jdbc:postgresql://localhost:5432/eduquizdev
+    url: jdbc:postgresql://localhost:5432/postgres
```

Its Your personal DB name?
```suggestion
    url: jdbc:postgresql://localhost:5432/eduquizdev
```